app/resources/v1/users.js

```diff
@@ -149,22 +180,20 @@ exports.get = function (req, res) {
           logger.error('Twitter API call users/show did not return any data.')
         }
 
-        sendUserJson(res)
+        sendUserJson({
+          twitter_profile_image_url: res.picture
+        })
       }
     } // END function - handleFetchUserProfileFromTwitter
 
-    if (twitterApiClient) {
+    if (twitterApiClient && !user.profile_image_url) {
       logger.debug('About to call Twitter API: /users/show.json?user_id=' + user.twitter_id)
       twitterApiClient.get('/users/show.json', { user_id: user.twitter_id }, handleFetchUserProfileFromTwitter)
-      setTimeout(
-        function () {
-          if (!responseAlreadySent) {
-            logger.debug('Timing out Twitter API call after %d milliseconds and sending partial response.', config.twitter.timeout_ms)
-            responseAlreadySent = true
-            sendUserJson()
-          }
-        },
-        config.twitter.timeout_ms)
+      if (!responseAlreadySent) {
+        logger.debug(`Timing out Twitter API call after %d milliseconds and sending partial response.`, config.twitter.timeout_ms)
+        responseAlreadySent = true
+        sendUserJson()
+      }
```

Comment:
Why was the `setTimeout` removed?

Reply:
Oh! that was a mistake I forgot to correct